src/types/TableTypes.ts

```diff
@@ -0,0 +1,15 @@
+export interface User {
+  name: string;
+  apartment: string;
+  tower: string;
+  rent: number;
+  moveInDate: string;
+  lastSignedIn: string;
+  actions: string[];
+}
+
+export interface TableProps {
+  headers: string[];
+  hasImage?: boolean;
```

Comment:
There is no need of hasImage prop because icon will be always present for tenants table